### DISCUSSION TITLE
Skipping Doc custom extension 'trf_data' while merging docs

### DISCUSSION THREAD

**Discussion by Onyoursix** [explosion/spaCy#7317] [Help: Coding & Implementations] [answered]
I'm batch processing some large documents via https://spacy.io/api/doc#from_docs

I'm receiving a warning

`/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:10: UserWarning: [W101] Skipping Doc custom extension 'trf_data' while merging docs.
 Remove the CWD from sys.path while we load stuff.`

Is this anything to be concerned about?

**Comment by adrianeboyd** [comment, marked as answer]
The `trf_data` (https://spacy.io/api/transformer#transformerdata) is used when the transformer pipeline is running, but you may not need it afterwards. Sometimes users want to reference this later so we save it in the doc, but if you only need the linguistic annotations on your docs when merging, then you can safely ignore this warning.

`Doc.from_docs` does try to preserve as much user data (custom extensions) as it can, so token or span extensions are preserved, but there's no good general-purpose way to merge doc-level data like `trf_data`, so it's skipped.
